test/type/string_test.exs

```diff
@@ -0,0 +1,120 @@
+defmodule Litmus.Type.StringTest do
+  use ExUnit.Case
+
+  alias Litmus.Type
+
+  describe "Type.validate/3" do
```

Comment:
What are your thoughts on creating a `type_test.exs` file and putting these kinds of tests in there instead?